installed-versions.json

```diff
@@ -1,14 +1,20 @@
 {
+  "version": "6.0.0",
```

Comment:
I don't recall this being here, or if we have a process to make sure it shows the correct version at release time. Maybe it doesn't matter much unless it shows in Sentry logs or AWS logs.

Reply:
It has been in the package.json file, but didn't ever get updated until we installed sentry. I suppose we could add it to a checklist, maybe in the PR template (which doesn't exist yet)